### ISSUE TITLE
[BUG] `at_start` called twice for global scripts on reload

### ISSUE THREAD

**Issue by owllex** [evennia/evennia#2609]
#### Describe the bug
When reloading the game with the reload command, global scripts that are defined in settings.GLOBAL_SCRIPTS have their `at_start` called twice. See stack traces below.

The bug does not occur for scripts that are not defined in GLOBAL_SCRIPTS.

#### To Reproduce
Steps to reproduce the behavior (observed in master branch):
1. Define a global script in settings.GLOBAL_SCRIPTS. Add a trackback log to its `at_start` method.
2. Run reload command
3. Observe two calls.

#### Expected behavior
`at_start` is called only once for scripts at startup regardless of how the script is defined.

#### Environment, Evennia version, OS etc
Evennia 0.9.5 (rev aa6d01e01) (rev aa6d01e01)
OS: posix (Note: WSL2)
Python: 3.8.10
Twisted: 21.7.0
Django: 3.2.8

#### Additional context
Did not test on develop.

<details>

<summary>Stack traces and system log</summary
<p>

```
22-01-07 17:00:36 [..] Server disconnected from the portal.
22-01-07 17:00:36 Main loop terminated.
22-01-07 17:00:36 Server Shut Down.
  File "/home/rel/mud/env/bin/twistd", line 8, in <module>
    sys.exit(run())
  File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/scripts/twistd.py", line 36, in run
    app.run(runApp, ServerOptions)
  File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/app.py", line 676, in run
    runApp(config)
  File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/scripts/twistd.py", line 30, in runApp
    runner.run()
  File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/app.py", line 372, in run
    self.application = self.createOrGetApplication()
  File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/app.py", line 439, in createOrGetApplication
    application = getApplication(self.config, passphrase)
  File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/app.py", line 448, in getApplication
    application = service.loadApplication(filename, style, passphrase)
  File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/service.py", line 404, in loadApplication
    application = sob.loadValueFromFile(filename, "application")
  File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/persisted/sob.py", line 176, in loadValueFromFile
    eval(codeObj, d, d)
  File "/home/rel/mud/evennia/evennia/server/server.py", line 26, in <module>
    evennia._init()
  File "/home/rel/mud/evennia/evennia/__init__.py", line 369, in _init
    GLOBAL_SCRIPTS.start()
  File "/home/rel/mud/evennia/evennia/utils/containers.py", line 182, in start
    self._load_script(key)
  File "/home/rel/mud/evennia/evennia/utils/containers.py", line 163, in _load_script
    found.restart(interval=interval, start_delay=start_delay, repeats=repeats)
  File "/home/rel/mud/evennia/evennia/scripts/scripts.py", line 599, in restart
    self.start()
  File "/home/rel/mud/evennia/evennia/scripts/scripts.py", line 477, in start
    self.at_start()
  File "/home/rel/mud/aelandris/typeclasses/weather.py", line 92, in at_start
    traceback.print_stack()
22-01-07 17:00:36 Loading /home/rel/mud/evennia/evennia/server/server.py...
22-01-07 17:00:36 Loaded.
22-01-07 17:00:36 twistd 21.7.0 (/home/rel/mud/env/bin/python 3.8.10) starting up.
22-01-07 17:00:36 reactor class: twisted.internet.epollreactor.EPollReactor.
22-01-07 17:00:36 Webserver starting on 4005
22-01-07 17:00:36 Server successfully reloaded.
22-01-07 17:00:36   File "/home/rel/mud/env/bin/twistd", line 8, in <module>
22-01-07 17:00:36     sys.exit(run())
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/scripts/twistd.py", line 36, in run
22-01-07 17:00:36     app.run(runApp, ServerOptions)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/app.py", line 676, in run
22-01-07 17:00:36     runApp(config)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/scripts/twistd.py", line 30, in runApp
22-01-07 17:00:36     runner.run()
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/app.py", line 376, in run
22-01-07 17:00:36     self.postApplication()
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/scripts/_twistd_unix.py", line 269, in postApplication
22-01-07 17:00:36     self.startReactor(None, self.oldstdout, self.oldstderr)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/app.py", line 388, in startReactor
22-01-07 17:00:36     runReactorWithLogging(self.config, oldstdout, oldstderr, self.profiler, reactor)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/application/app.py", line 306, in runReactorWithLogging
22-01-07 17:00:36     reactor.run()
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/internet/base.py", line 1318, in run
22-01-07 17:00:36     self.mainLoop()
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/internet/base.py", line 1331, in mainLoop
22-01-07 17:00:36     reactorBaseSelf.doIteration(t)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/internet/epollreactor.py", line 244, in doPoll
22-01-07 17:00:36     log.callWithLogger(selectable, _drdw, selectable, fd, event)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/python/log.py", line 101, in callWithLogger
22-01-07 17:00:36     return callWithContext({"system": lp}, func, *args, **kw)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/python/log.py", line 85, in callWithContext
22-01-07 17:00:36     return context.call({ILogContext: newCtx}, func, *args, **kw)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/python/context.py", line 118, in callWithContext
22-01-07 17:00:36     return self.currentContext().callWithContext(ctx, func, *args, **kw)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/python/context.py", line 83, in callWithContext
22-01-07 17:00:36     return func(*args, **kw)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/internet/posixbase.py", line 686, in _doReadOrWrite
22-01-07 17:00:36     why = selectable.doRead()
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/internet/tcp.py", line 247, in doRead
22-01-07 17:00:36     return self._dataReceived(data)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/internet/tcp.py", line 252, in _dataReceived
22-01-07 17:00:36     rval = self.protocol.dataReceived(data)
22-01-07 17:00:36   File "/home/rel/mud/evennia/evennia/server/portal/amp.py", line 364, in dataReceived
22-01-07 17:00:36     super(AMPMultiConnectionProtocol, self).dataReceived(data)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/protocols/amp.py", line 2360, in dataReceived
22-01-07 17:00:36     return Int16StringReceiver.dataReceived(self, data)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/protocols/basic.py", line 731, in dataReceived
22-01-07 17:00:36     self.stringReceived(packet)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/protocols/basic.py", line 841, in stringReceived
22-01-07 17:00:36     self.state = statehandler(string)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/protocols/amp.py", line 2411, in proto_key
22-01-07 17:00:36     self.boxReceiver.ampBoxReceived(self._currentBox)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/protocols/amp.py", line 1031, in ampBoxReceived
22-01-07 17:00:36     self._commandReceived(box)
22-01-07 17:00:36   File "/home/rel/mud/evennia/evennia/server/portal/amp.py", line 347, in _commandReceived
22-01-07 17:00:36     deferred = self.dispatchCommand(box)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/protocols/amp.py", line 1065, in dispatchCommand
22-01-07 17:00:36     return maybeDeferred(responder, box)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/internet/defer.py", line 191, in maybeDeferred
22-01-07 17:00:36     result = f(*args, **kwargs)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/protocols/amp.py", line 1159, in doit
22-01-07 17:00:36     maybeDeferred(aCallable, **kw)
22-01-07 17:00:36   File "/home/rel/mud/env/lib/python3.8/site-packages/twisted/internet/defer.py", line 191, in maybeDeferred
22-01-07 17:00:36     result = f(*args, **kwargs)
22-01-07 17:00:36   File "/home/rel/mud/evennia/evennia/server/amp_client.py", line 230, in server_receive_adminportal2server
22-01-07 17:00:36     server_sessionhandler.portal_sessions_sync(kwargs.get("sessiondata"))
22-01-07 17:00:36   File "/home/rel/mud/evennia/evennia/server/sessionhandler.py", line 400, in portal_sessions_sync
22-01-07 17:00:36     self.server.at_post_portal_sync(mode)
22-01-07 17:00:36   File "/home/rel/mud/evennia/evennia/server/server.py", line 537, in at_post_portal_sync
22-01-07 17:00:36     ScriptDB.objects.validate(init_mode=mode)
22-01-07 17:00:36   File "/home/rel/mud/evennia/evennia/scripts/manager.py", line 207, in validate
22-01-07 17:00:36     nr_started += script.start(force_restart=init_mode)
22-01-07 17:00:36   File "/home/rel/mud/evennia/evennia/scripts/scripts.py", line 477, in start
22-01-07 17:00:36     self.at_start()
22-01-07 17:00:36   File "/home/rel/mud/aelandris/typeclasses/weather.py", line 92, in at_start
22-01-07 17:00:36     traceback.print_stack()
```

</p>
</details>
